besu/src/test/java/org/hyperledger/besu/PrivacyReorgTest.java

```diff
@@ -368,8 +367,7 @@ public void reorgToLongerChain() {
     appendBlock(besuController, blockchain, protocolContext, thirdForkBlock);
 
     // Check that the private state did change after reorg
-    assertPrivateStateRoot(
-        privateStateRootResolver, blockchain, STATE_ROOT_AFTER_TRANSACTION_APPENDED_TO_EMPTY_STATE);
```

Comment:
@lucassaldanha does this make sense? We're testing that the state root hash is `EMPTY_ROOT_HASH` still.

Reply:
I think we might need to take a look at this. My understanding is that at the end, we are adding a 3rd block to the new chain, and that the block contains a PMT.Therefore, I would expect the private state root hash to be different from the empty state root hash (assuming this private tx in the 3rd block was processed and altered the state).

I'll ask someone from the team to take a look at it.

Reply:
Ok cool, I merged because it had bugfixes important for 1559 but let me know if there's anything more that needs to be done with the test.